### PR TITLE
docs: mf dev cli commands

### PR DESCRIPTION
**Pull request by Timeless0911**
## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

File: website/docs/en/guide/basic/cli.mdx [hunk 1]
```diff
@@ -105,12 +106,13 @@ Inspect config succeed, open following files to view the content:
 ## rslib mf dev
 
 The `rslib mf dev` command is utilized to start Rsbuild dev server of Module Federation format.
+
 This enables you to develop your mf format module within the host app more rapidly.
 
 ```text
-Usage: rslib mf dev [options]
```
Comment (nyqykk) [inline website/docs/en/guide/basic/cli.mdx:111]: Should we add options before specific command ？

Comment (nyqykk) [reply website/docs/en/guide/basic/cli.mdx:111]: rslib mf dev —xxx or rslib mf —xxx dev ？

File: website/docs/en/guide/basic/cli.mdx [hunk 2]
```diff
@@ -105,12 +106,13 @@ Inspect config succeed, open following files to view the content:
...
+Usage: rslib mf [options] <dev>
```
[the remark this answers is not in the document]
Comment (fi3ework) [reply website/docs/en/guide/basic/cli.mdx:113]: And, put sub commands ahead (`rslib mf dev --lib mf0`) is more common than `rslib mf --lib mf0 dev`?

[the remark this answers is not in the document]
Comment (fi3ework) [reply website/docs/en/guide/basic/cli.mdx:113]: LOL.

**Review by fi3ework — APPROVED** [review]

**Pull request merged**